### ISSUE TITLE
Build works on Linux, but tests return 7 errors with latest commit (no 145, or 1865d2f), but didn't with commit 144

### ISSUE THREAD

**Issue by fusion809** [RAunplugged/uRA#47]
Hi,

On Arch Linux, among other Linux distros (including CentOS, Fedora, openSUSE and Scientific Linux; for details see where I build this fine mod in the openSUSE Build Service &mdash; https://build.opensuse.org/package/show/home:fusion809/openra-ura), `make` runs fine but `make test` fails with 7 errors, they seem to come from rules.yaml and rules/structures.yaml. The full log is: [test.log](https://github.com/RAunplugged/uRA/files/2364164/test.log). 

Thanks for your time,
Brenton

**Comment by fusion809** [comment]
(Didn't realize there were two commits in quick succession today, so it turns out I hadn't tried 144).

**Comment by fusion809** [comment]
Oh and I can confirm this also happens on Linux Mint 19.

**Comment by fusion809** [comment]
Same issue with commit 147 (39759507ad10dad66c2bf326bca71f91322fd385).

**Comment by fusion809** [comment]
Commit 154 (606f0f4decfda0fad80e90ceb8d47a25c714b1e6) seems to have made the number of errors greater (changing from 7 to 9): [here](https://github.com/RAunplugged/uRA/files/2364721/test.log)'s the present log.

**Comment by SoScared** [comment]
This is getting interesting, thanks for the quick report!

There are a few errors I should be able to fix quickly that relates to my recent commits.

**Comment by SoScared** [comment]
Ok, fixed the errors reported by TravisCI at least.

**Comment by fusion809** [comment]
Yep builds on the OBS are fixed too (https://build.opensuse.org/package/show/home:fusion809/openra-ura), aside from openSUSE ones that seem trivial to fix (ignore the Arch one too as it's an error that occurs on all my Arch builds).

**Comment by SoScared** [comment]
Great Success!

OpenRA still grapples with their new Linux install features, if lucky those issues are solved with the engine.
